raiden_contracts/tests/conftest.py

```diff
@@ -1,2 +1,9 @@
-# pylint: disable=W0614, C0111, W0401
+"""
+isort:skip_file
+"""
+from pytest import register_assert_rewrite
+
+register_assert_rewrite("raiden_contracts.utils.events")
+register_assert_rewrite("raiden_contracts.tests.fixtures.channel")
```

Comment:
from [here](https://docs.pytest.org/en/stable/writing_plugins.html#assertion-rewriting) it looks as if you could give an iterable